App/App.xaml.cs

```diff
@@ -190,7 +194,18 @@ public void OnActivated(object? sender, AppActivationArguments args)
                     _logger.LogWarning("URI activation with null data");
                     return;
                 }
-                HandleURIActivation(protoArgs.Uri);
+
+                try
+                {
+                    // don't need to wait for it to complete.
+                    _ = _uriHandler.HandleUri(protoArgs.Uri);
+                }
+                catch (System.Exception e)
+                {
+                    _logger.LogError(e, "unhandled exception while processing URI coder://{authority}{path}",
+                        protoArgs.Uri.Authority, protoArgs.Uri.AbsolutePath);
+                }
```

Comment:
This would probably be better as a `Task.ContinueWith()` call to log the exception